src/ontology/pato-edit.obo

```diff
@@ -21856,6 +21856,66 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070015
+name: pyramidal family morphology
+def: "A cell morphology that inheres in neurons which have a pyramidal shaped soma, a single axon, a large apical dendrite and multiple basal dendrites." [doi:10.1016/b978-0-12-369497-3.10004-4]
+xref: ILX:0109553
+xref: SAO:1465673360
+is_a: PATO:0010006 ! cell morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070016
+name: standard pyramidal morphology
+def: "A pyramidal cell morphology that inheres in neurons which have a standard pyramidal shape with its apical dendrite aligned vertically." [doi:10.1016/b978-0-12-369497-3.10004-4]
+synonym: "pyramidal morphology" EXACT []
+synonym: "vertical pyramidal morphology" EXACT []
+xref: ilxtr:PyramidalPhenotype
+is_a: PATO:0070015 ! pyramidal family morphology
+disjoint_from: PATO:0070017 ! horizontal pyramidal morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070017
+name: horizontal pyramidal morphology
+def: "A pyramidal cell morphology that inheres in neurons which have an apical tree which is oriented parallel to the pia." [PMID:30715238]
+xref: ilxtr:HorizontalPyramidalPhenotype
+is_a: PATO:0070015 ! pyramidal family morphology
+created_by: http://orcid.org/0000-0001-7258-9596
+
+[Term]
+id: PATO:0070018
+name: tufted pyramidal morphology
+def: "A pyramidal cell morphology that inheres in neurons which have a distinct tuft formation, distal from the soma." [PMID:30715238]
+xref: ilxtr:TuftedPyramidalPhenotype
+is_a: PATO:0070015 ! pyramidal family morphology
```

Comment:
Is this the correct hierarchy? I thought 'pyramidal family morphology' was supposed to be the superclass with 'standard pyramidal morphology' and 'horizontal pyramidal morphology' as disjoint subclasses (as you have it here, correctly, I think). But my understanding from Tom's comments were that 'tufted pyramidal morphology', 'untufted pyramidal morphology', 'stellate pyramidal morphology', and 'inverted pyramidal morphology' were subclasses of 'standard pyramidal morphology' not direct children of 'pyramidal family morphology', which is a helper class of sorts.  

Also (small issue) I think it's standard to have the parent term as the genus in the textual definition, but I don't see 'pyramidal cell morphology' as a parent term here. Maybe this is lending to my confusion. If this is confusing to you, I'm happy to add line comments for each change that you can accept or reject accordingly.

Reply:
Oh, sorry, totally misunderstood, though I wonder if there are horizontal pyramidal that are also considered tufted etc? Do need input on this. 

>Also (small issue) I think it's standard to have the parent term as the genus in the textual definition, but I don't see 'pyramidal cell morphology' as a parent term here.

oh yeah, that was me forgetting to change it to 'pyramidal family morphology' - but I guess it does bring up this disconnect in understanding. Will change accordingly once we get some input from @tgbugs, thanks for pointing it out @patrick-lloyd-ray 

>I'm happy to add line comments for each change that you can accept or reject accordingly.

no need for this, will make all the changes once we figure out whats the best way to approach :) thanks!